code/datums/mood_events/needs_events.dm

```diff
@@ -91,3 +91,40 @@
 	description = "There's nothing like the feeling of a freshly laundered jumpsuit."
 	mood_change = 2
 	timeout = 10 MINUTES
+
+
+//nutrition
+/datum/mood_event/ate_event
+	timeout = 10 MINUTES
+
+/datum/mood_event/ate_event/no_table
+	description = "I had to eat a meal off the ground. Can't we get a table around here?"
+	mood_change = -5
```

Comment:
lol

Reply:
especially considering these stack, do not make eating without a table, chair, and forks make you instantly fucking depressed/insane

Reply:
It doesn't it takes 5 minutes for negative moods to impact sanity

Reply:
okay, then 

especially considering these stack, do not make eating without a table, chair, and forks make you fucking depressed/insane after 5 minutes

Reply:
we can adjust the numbers after testing

Reply:
No, that is not an excuse--we know how -5 interacts, and we know how it'll interact with stacking